src/include/optimizer/index_util.h

```diff
@@ -59,10 +60,13 @@ class IndexUtil {
    * @param prop PropertySort to satisfy
    * @param tbl_oid OID of the table that the index is built on
    * @param idx_oid OID of index to use to satisfy
+   * @param bounds bounds for IndexScan
    * @returns TRUE if the specified index can fulfill sort property
    */
-  static bool SatisfiesSortWithIndex(catalog::CatalogAccessor *accessor, const PropertySort *prop,
-                                     catalog::table_oid_t tbl_oid, catalog::index_oid_t idx_oid);
+  static bool SatisfiesSortWithIndex(
+      catalog::CatalogAccessor *accessor, const PropertySort *prop, catalog::table_oid_t tbl_oid,
+      catalog::index_oid_t idx_oid,
+      std::unordered_map<catalog::indexkeycol_oid_t, std::vector<planner::IndexExpression>> *bounds = nullptr);
```

Comment:
Could you add a comment for this ? - did u run doxygen on this? - it should have yelled